crates/biome_css_parser/tests/css_test_suite/error/function/if.css.snap

```diff
@@ -0,0 +1,3570 @@
+              0: DOT@1299..1302 "." [Newline("\n"), Newline("\n")] []
+              1: CSS_CUSTOM_IDENTIFIER@1302..1311
+                0: IDENT@1302..1311 "empty-if" [] [Whitespace(" ")]
+      1: CSS_DECLARATION_OR_RULE_BLOCK@1311..1328
+        0: L_CURLY@1311..1312 "{" [] []
+        1: CSS_DECLARATION_OR_RULE_LIST@1312..1326
+          0: CSS_DECLARATION_WITH_SEMICOLON@1312..1326
+            0: CSS_DECLARATION@1312..1325
+              0: CSS_GENERIC_PROPERTY@1312..1325
+                0: CSS_IDENTIFIER@1312..1319
+                  0: IDENT@1312..1319 "color" [Newline("\n"), Whitespace("\t")] []
+                1: COLON@1319..1321 ":" [] [Whitespace(" ")]
+                2: CSS_GENERIC_COMPONENT_VALUE_LIST@1321..1325
+                  0: CSS_IF_FUNCTION@1321..1325
+                    0: IF_KW@1321..1323 "if" [] []
+                    1: L_PAREN@1323..1324 "(" [] []
+                    2: CSS_IF_BRANCH_LIST@1324..1324
+                      0: CSS_IF_BRANCH@1324..1324
+                        0: (empty)
+                        1: (empty)
+                        2: CSS_GENERIC_COMPONENT_VALUE_LIST@1324..1324
+                    3: R_PAREN@1324..1325 ")" [] []
+              1: (empty)
+            1: SEMICOLON@1325..1326 ";" [] []
+        2: R_CURLY@1326..1328 "}" [Newline("\n")] []
+  2: EOF@1328..1329 "" [Newline("\n")] []
+
+```
+
+## Diagnostics
+
+```
+if.css:2:30 parse ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+
+  × expected `:` but instead found `black`
+  
+    1 │ .missing-colon {
+  > 2 │ 	color: if(style(--foo: bar) black);
+      │ 	                            ^^^^^
+    3 │ }
+    4 │ 
+  
+  i Remove black
+  
+if.css:6:12 parse ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+
+  × Expected an if branch, or an if test boolean expression but instead found ':'.
+  
+    5 │ .empty-condition {
+  > 6 │ 	color: if(: black);
+      │ 	          ^
+    7 │ }
+    8 │ 
+  
+  i Expected an if branch, or an if test boolean expression here.
+  
+    5 │ .empty-condition {
+  > 6 │ 	color: if(: black);
+      │ 	          ^
+    7 │ }
+    8 │ 
+  
+if.css:10:28 parse ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+
+  × Unexpected value or character.
+  
+     9 │ .unclosed-style {
+  > 10 │ 	color: if(style(--foo: bar: black);
+       │ 	                          ^
+    11 │ }
+    12 │ 
+  
+  i Expected one of:
+  
+  - identifier
+  - string
+  - number
+  - dimension
+  - ratio
+  - custom property
+  - function
+  
+if.css:14:22 parse ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+
+  × Unexpected value or character.
+  
+    13 │ .invalid-style-query {
+  > 14 │ 	color: if(style(not a valid query): black);
+       │ 	                    ^^^^^^^^^^^^^
+    15 │ }
+    16 │ 
+  
+  i Expected one of:
+  
+  - ( <style-query> )
+  - ( <style-feature> )
```

Comment:
These are not formatted like other diagnostics are.

Reply:
I think these are inherited from reuse of the style in container nodes ([similar existing example here](https://github.com/ryan-m-walker/biome/blob/main/crates/biome_css_parser/tests/css_test_suite/error/at_rule/at_rule_container/at_rule_container_style_not_query.css.snap#L573-L574), coming from [here](https://github.com/ryan-m-walker/biome/blob/main/crates/biome_css_parser/src/syntax/at_rule/container/error.rs#L54)).